modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -175,3 +175,9 @@ You receive an email from Anypoint Platform with a list of user names associated
 == Upgrade Your Account
 
 To upgrade your Anypoint Platform trial account to an enterprise license, contact your account representative or https://www.mulesoft.com/contact[the MuleSoft sales team].
+
+== Known Issue: Account Creation
+
+When you attempt to create an account from an email invitation, an error appears, erroneously stating your credentials are invalid.
+This error occurs when a username already exists, even if it is under a different organization.
+If the username already belongs to you, you can move the username from your previous organization to the organization that sent you the email invitation.
```

Comment:
"If the username belongs to you in another organization, you can move the name to the organization that sent you the email invitation."

What if the username is already being used by someone else?

Reply:
They'd have to pick another. If I'm sarathecoubian in MuleSoft and I click a link to join Salesforce, I would encounter this issue and would need my username to be moved.